src/cr/cube/matrix/assembler.py

```diff
@@ -233,47 +233,64 @@ def means(self):
         """
         return self._assemble_matrix(self._measures.means.blocks)
 
-    def pairwise_indices(self, alpha, only_larger):
+    @lazyproperty
+    def pairwise_indices(self):
         """2D optional ndarray of tuple of int column-idxs pairwise-t threshold.
 
         Raises `ValueError if the cube-result does not include `overlaps` cube-measures.
         """
-        if self._cube_has_overlaps:
-            # If overlaps are defined, calculate significance based on them
-            return self._assemble_matrix(
-                self._measures.pairwise_indices_for_subvar(alpha, only_larger).blocks
-            )
+        p_vals, t_stats = (
+            self._measures.pairwise_indices_for_subvar.values
+            if self._cube.has_overlaps
+            else self._measures.pairwise_indices.values
+        )
+        return self._assemble_pw_indices(
+            p_vals, t_stats, self._alpha, self._only_larger
+        )
 
-        return np.array(
-            [
-                self._pairwise_indices(
-                    self.pairwise_significance_p_vals(col),
-                    self.pairwise_significance_t_stats(col),
-                    alpha,
-                    only_larger,
-                )
-                for col in range(len(self._column_order))
-            ]
-        ).T
+    @lazyproperty
+    def pairwise_indices_alt(self):
+        """2D optional ndarray of tuple of int column-idxs pairwise-t alt. threshold.
```

Comment:
"Optional 2D ndarray", not 2D optional ndarray.

The " pairwise-t alt. threshold" suffix doesn't make any sense. Just end the summary sentence after "column_idxs." and add a continuation paragraph where you spell out the rest.

Also this one makes the same overgeneralization of when `ValueError` might be raised.

While we're talking about that, is that exception still possible and if so who is catching that exception? I'm not sure we want to propagate that to users, do we?